hawkular-inventory-bus/src/main/java/org/hawkular/inventory/bus/MessageSender.java

```diff
@@ -65,4 +73,18 @@ public void send(Interest<?, ?> interest, Object inventoryEvent) {
     private String firstLetterLowercased(String source) {
         return Character.toLowerCase(source.charAt(0)) + source.substring(1);
     }
+
+    private ProducerConnectionContext init() throws JMSException, NamingException {
+        InitialContext namingContext = new InitialContext();
```

Comment:
I wanted to comment don't merge yet. But you were so fat.  Yes it's not efficient to reinitialize it every time message is sent... 

I will correct when I get home or in the morning.